src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -14,7 +18,11 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.QuickInfo
 {
     internal static class IntellisenseQuickInfoBuilder
     {
-        internal static IntellisenseQuickInfoItem BuildItem(ITrackingSpan trackingSpan, CodeAnalysisQuickInfoItem quickInfoItem)
+        internal static async Task<IntellisenseQuickInfoItem> BuildItemAsync(ITrackingSpan trackingSpan,
+            CodeAnalysisQuickInfoItem quickInfoItem,
+            ITextSnapshot snapshot,
+            Document document,
+            CancellationToken cancellationToken)
```

Comment:
@jasonmalinowski were tehre any threading concerns here?  Is it safe to do ConfigureAwait(false)'s here?

Reply:
Generally I'd be expecting `ConfigureAwait(false)` here: this is going from one data model to another.

Reply:
Ok thanks.  I wasn't sure about ClassifiedTextElement  and if that had any sort of ThreadAffinity concerns.

Reply:
Nope, it's basically a list of (string, string) tuples and is thread-safe.